pkg/prommetrics/prommetrics.go

```diff
@@ -204,3 +216,23 @@ func DecrementCRDTotal(crdType, namespace string) {
 
 	crdTotalsGaugeVec.WithLabelValues(crdType, namespace).Dec()
 }
+
+func RecordScalerPausedInc(namespace string, scaledObject string) {
+	labels := prometheus.Labels{"namespace": namespace, "scaledObject": scaledObject}
+
+	scaledObjectPausedTotal.With(labels).Inc()
+}
+func RecordScalerPausedDec(namespace string, scaledObject string) {
+	labels := prometheus.Labels{"namespace": namespace, "scaledObject": scaledObject}
+
+	scaledObjectPausedTotal.With(labels).Dec()
+}
+
+func GetObjectPausedValue(namespace string, scaledObject string) float64 {
+	var metric = &dto.Metric{}
+	if err := scaledObjectPausedTotal.WithLabelValues(namespace, scaledObject).Write(metric); err != nil {
+		return 0
+	}
+
+	return metric.Gauge.GetValue()
+}
```

Comment:
Instead of using this logic, we could just set 1 or 0 ([as we do here for example](https://github.com/kedacore/keda/blob/main/pkg/prommetrics/prommetrics.go#L128)). 
The metric has labels to make it unique, so I think that we can just set 1 or 0 on every loop and that avoid the requirement of querying the previous stored value.
WDYT?

Reply:
make sense. fixed